package.json

```diff
@@ -58,7 +50,8 @@
     "test-coverage": "node ./out/test/runTest.js --coverage",
     "prettier:write": "prettier --write \"src/**/*.{ts, js}\"",
     "prettier:verify": "prettier --list-different \"src/**/*.{ts, js}\"",
-    "vscode:prepublish": "npm run pretest && npm run test"
+    "bundle:extension": "esbuild ./src/extension.ts  --bundle --outdir=out --format=cjs --target=es2020 --platform=node --external:vscode --external:@salesforce/core --external:@salesforce/lwc-dev-mobile-core --external:@salesforce/sf-plugins-core --minify",
```

Comment:
Ah, let me try adding it. I have gone through so many iterations of the esbuild commands, not sure how I arrived at this one, but other than the `--sourcemap`, looks close unless I'm missing something?

Reply:
It looks pretty much the same as the example, otherwise. I like the way the example in the docs is split out, because it also provides a target for being able to debug. But we can also cover that split-out in our next iteration of bundling to come. I'm okay with it as we've coded it today. Presumably `--sourcemap` won't negatively impact what we have here.

Reply:
Oh I just noticed I looked at it wrong! In the example configuration, `vscode:prepublish` calls the `esbuild-base` script, which does _not_ specify `--sourcemap`. The `esbuild` script—which is not otherwise called in the publishing workflow—calls `esbuild-base`, but _with_ the `--sourcemap` argument added on.

So presumably `--sourcemap` not necessary for actual publishing, and is probably more useful in local use cases?

Reply:
I understand it is more for generating the mapping from minified to debug code, similar to a dsym on iOS or mapping.txt for Android proguard obfuscation? It is being generated, but not bundled in the VSIX currently.

Reply:
Exactly: it's meant to tie back to code _in a development environment_, to facilitate stepping through minified code, or JS code transpiled from TypeScript, etc. It's not required for runtime functionality, just for debugging it. So they don't have any real purpose in your final bundle. 👍